packages/concerto-core/lib/decoratormanager.js

```diff
@@ -253,6 +253,233 @@ class DecoratorManager {
         return newModelManager;
     }
 
+    /**
+     * Extracts all the decorator commands from all the models in modelManager
+     * @param {ModelManager} modelManager the input model manager
+     * @param {object} options - decorator models options
+     * @param {boolean} options.removeDecoratorsFromModel - flag to strip out decorators from models
+     * @param {string} options.locale - locale for extracted vocabulary set
+     * @returns {Object} a new model manager with the decorations removed and a list of extracted decorator jsons
+     */
+    static extractDecorators(modelManager,options) {
+        options = {
+            ...{
+                removeDecoratorsFromModel: false,
+                locale:'en'
+            },
+            ...options || {}
+        };
```

Comment:
Is this equivalent to:
```suggestion
        options = {
            removeDecoratorsFromModel: false,
            locale:'en',
            ...options
        };
```

Reply:
If the options is not provided, then it would fail the unpacking.

Reply:
My bad, it doesn't fail.